packages/jest/rules-snapshot.json

```diff
@@ -40,18 +40,16 @@
   "jest/no-standalone-expect": "error",
   "jest/no-test-prefixes": "error",
   "jest/no-test-return-statement": "error",
-  "jest/no-try-expect": "error",
   "jest/prefer-hooks-on-top": "error",
   "jest/prefer-spy-on": "error",
   "jest/prefer-strict-equal": "error",
-  "jest/prefer-to-be-null": "error",
-  "jest/prefer-to-be-undefined": "error",
+  "jest/prefer-to-be": "error",
   "jest/prefer-to-contain": "error",
   "jest/prefer-to-have-length": "error",
   "jest/prefer-todo": "error",
   "jest/require-to-throw-message": "error",
   "jest/require-top-level-describe": "error",
-  "jest/valid-describe": "error",
+  "jest/valid-describe-callback": "error",
```

Comment:
This is just a non-functional rename: https://github.com/jest-community/eslint-plugin-jest/commit/f3e9e9a64e183a0fb8af3436611a7f70366a528d